web_widget_image_download/static/src/js/web_widget_image_download.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2016 Flavio Corpa <flavio.corpa@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+ */
+openerp.web_widget_image_download = function(instance) {
+    'use strict';
+
+    instance.web.form.web_widget_image_download = instance.web.form.FieldBinaryImage.include({
+        render_value() {
+            this._super();
+
+            // remove `col-md-offset-5` from following trash icon (improves visibility)
+            this.$el.find('.oe_form_binary_file_clear').removeClass('col-md-offset-5');
```

Comment:
QWeb Engine in JS side doesn't support attributes position. In this case, we must use replace. And I don't like replaces because they could break inheritance. In this case I think that JQuery manipulation is the safest way.